tests/test_search.py

```diff
@@ -1,60 +1,201 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+import datetime
 import json
 import logging
 import os
 import sys
+import time
 import unittest
+import uuid
 
 import requests
 
-pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
 import dss
+from dss import DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE
 from dss.events.handlers.index import create_elasticsearch_index
-from dss.util.es import ElasticsearchServer
-from tests.infra import DSSAsserts
+from dss.util.es import ElasticsearchServer, ElasticsearchClient
+from tests.infra import DSSAsserts, start_verbose_logging
+from tests.es import elasticsearch_delete_index
+from tests import smartseq2_paired_ends_query, get_version
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+start_verbose_logging()
 
 
 class TestSearch(unittest.TestCase, DSSAsserts):
     @classmethod
     def setUpClass(cls):
         cls.es_server = ElasticsearchServer()
         os.environ['DSS_ES_PORT'] = str(cls.es_server.port)
+        dss.Config.set_config(dss.DeploymentStage.TEST)
+        cls.app = dss.create_app().app.test_client()
+        with open(os.path.join(os.path.dirname(__file__), "sample_index_doc.json"), "r") as fh:
+            cls.index_document = json.load(fh)
+
+    @classmethod
+    def load_sample_document(cls, file_name):
+        with open(os.path.join(os.path.dirname(__file__), file_name), "r") as fh:
+            return json.load(fh)
 
     @classmethod
     def tearDownClass(cls):
         cls.es_server.shutdown()
 
     def setUp(self):
-        dss.Config.set_config(dss.DeploymentStage.TEST)
-        self.app = dss.create_app().app.test_client()
-        create_elasticsearch_index(logging.getLogger(__name__))
+        elasticsearch_delete_index(DSS_ELASTICSEARCH_INDEX_NAME)
+        create_elasticsearch_index(logger)
 
     def test_search_post(self):
+        bundle_uuid = str(uuid.uuid4())
+        version = get_version()
+        self.index_document['manifest']['version'] = version
+        bundle_id = f"{bundle_uuid}.{version}"
+        bundle_url = f"http://localhost/v1/bundles/{bundle_uuid}?version={version}"
+
+        es_client = ElasticsearchClient.get(logger)
+        es_client.index(index=DSS_ELASTICSEARCH_INDEX_NAME,
+                        doc_type=DSS_ELASTICSEARCH_DOC_TYPE,
+                        id=bundle_id,
+                        body=self.index_document,
+                        refresh=True)
+
+        response = self.assertPostResponse(
+            "/v1/search",
+            json_request_body=smartseq2_paired_ends_query,
+            expected_code=requests.codes.ok)
+        search_response = response.json
+        self.assertEqual(len(search_response), 2)
```

Comment:
Still think you should remove this.  It has no bearing on the fields you care about.

Reply:
removed